backend/prisma/migrations/20240709190435_add_route_and_user_profile_fields/migration.sql

```diff
@@ -0,0 +1,116 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "username" TEXT NOT NULL,
+    "password" TEXT NOT NULL,
+    "age" INTEGER,
+    "gender" TEXT,
+    "weight" DOUBLE PRECISION,
+    "height" DOUBLE PRECISION,
+    "fitnessLevel" TEXT,
+    "runningExperience" TEXT,
+    "preferredTerrains" TEXT[],
+    "healthConditions" TEXT[],
+    "lastKnownLocation" TEXT,
+
+    CONSTRAINT "User_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Token" (
+    "id" SERIAL NOT NULL,
+    "refreshToken" TEXT NOT NULL,
+    "userId" INTEGER NOT NULL,
+
+    CONSTRAINT "Token_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Post" (
+    "id" SERIAL NOT NULL,
+    "title" TEXT NOT NULL,
+    "content" TEXT,
+    "userId" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "Post_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Image" (
+    "id" SERIAL NOT NULL,
+    "data" BYTEA NOT NULL,
+    "mimeType" TEXT NOT NULL,
+    "postId" INTEGER NOT NULL,
+
+    CONSTRAINT "Image_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "UserRunningGoal" (
+    "id" SERIAL NOT NULL,
+    "userId" INTEGER NOT NULL,
+    "goalType" TEXT NOT NULL,
+    "targetValue" DOUBLE PRECISION NOT NULL,
+    "deadline" TIMESTAMP(3),
+    "isCompleted" BOOLEAN NOT NULL DEFAULT false,
+
+    CONSTRAINT "UserRunningGoal_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Route" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "distance" DOUBLE PRECISION NOT NULL,
+    "elevationGain" DOUBLE PRECISION NOT NULL,
+    "terrainType" TEXT NOT NULL,
+    "startPoint" TEXT NOT NULL,
+    "endPoint" TEXT NOT NULL,
+    "waypoints" TEXT[],
+    "difficulty" TEXT NOT NULL,
+    "sceneryType" TEXT[],
+    "pointsOfInterest" TEXT[],
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
```

Comment:
just confirming but is it necessary to have updatedAt column in Route table? Will the Route be updated after it's been created?

Reply:
I currently have the updated at just in case for future cases in which Users might provide feedback or suggest changes to existing routes. However for now it is not necessary, therefore i will remove it

Reply:
If this is the case, feel free to leave it! Just wanted to make sure there was a purpose for why that column exists